es6/src/sample-module.js

```diff
@@ -1,6 +1,11 @@
 "use strict";
 const capitalizeString = (string) => {
-  string.charAt(0).toUpperCase() + string.slice(1);
+  if (typeof window === 'undefined') {
+    console.log('capitalizeString called!');
```

Comment:
alert (Window.alert) method is only available in browser. So, add conditional statement.